packages/sanes-browser-extension/src/routes/welcome/index.stories.tsx

```diff
@@ -2,14 +2,35 @@ import { storiesOf } from "@storybook/react";
 import { Storybook } from "medulas-react-components";
 import React from "react";
 
+import { PersonaProvider } from "../../context/PersonaProvider";
+import { GetPersonaResponse } from "../../extension/background/model/backgroundscript";
 import { CHROME_EXTENSION_ROOT } from "../../utils/storybook";
 import Layout from "./index";
 
-storiesOf(CHROME_EXTENSION_ROOT, module).add(
-  "Welcome page",
-  (): JSX.Element => (
-    <Storybook>
-      <Layout />
-    </Storybook>
-  ),
-);
+const persona: GetPersonaResponse = {
+  mnemonic: "",
+  accounts: [{ label: "Account 0" }],
+  txs: [],
+};
+
+storiesOf(`${CHROME_EXTENSION_ROOT}/Welcome page`, module)
+  .add(
+    "With persona",
+    (): JSX.Element => (
+      <PersonaProvider persona={persona} hasStoredPersona={true}>
+        <Storybook>
+          <Layout />
+        </Storybook>
+      </PersonaProvider>
+    ),
+  )
+  .add(
+    "Without persona",
+    (): JSX.Element => (
```

Comment:
Return types of unnames function not needed `() => (`, which makes storybook and test code much shorter